Aztec/Classes/TextKit/TextView.swift

```diff
@@ -224,7 +224,9 @@ open class TextView: UITextView {
     ///
     override open var typingAttributes: [String: Any] {
         get {
-            ensureRemovalOfParagraphAttributesAfterSelectionChange()
+            if markedTextRange == nil {
```

Comment:
Would it be possible to move this condition inside the `mustRemoveParagraphAttributesAfterSelectionChange` method?

That way the `typingAttributes` override would look the same, and all of the conditions would be contained as follows:

```
    private func mustRemoveParagraphAttributesAfterSelectionChange() -> Bool {
        return selectedRange.location == storage.length
            && storage.string.isEmptyParagraph(at: selectedRange.location)
            && markedTextRange == nil
    }
```